esphome/components/sdp3x/sensor.py

```diff
@@ -0,0 +1,32 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c, sensor
+from esphome.const import (
+    CONF_ID,
+    DEVICE_CLASS_PRESSURE,
+    ICON_EMPTY,
+    UNIT_HECTOPASCAL,
+)
+
+DEPENDENCIES = ["i2c"]
```

Comment:
Please add yourself as a codeowner and run `script/build_codeowners.py`
```suggestion
CODEOWNERS = ["@Azimath"]
DEPENDENCIES = ["i2c"]
```

Reply:
Done, github doesn't see it as outdated for some reason